Bridges/HttpKernel.php

```diff
@@ -110,6 +110,10 @@ public function onRequest(ReactRequest $request, HttpResponse $response)
         if ($this->application instanceof TerminableInterface) {
             $this->application->terminate($syRequest, $syResponse);
         }
+	
+	    if (is_a($this->application, '\Illuminate\Contracts\Http\Kernel')) {
```

Comment:
For this specifical part I copied the implementation from #39 and checked it with xdebug that it was working correctly. If for some reason you prefer instanceof (they should be equivalent in the effects, but probably instanceof is faster since is a language construct and not a function) I can test it in the next few days and update the implementation